### DISCUSSION TITLE
fake ip模式下qBittorrent无法连接

### DISCUSSION THREAD

**Discussion by kid-oh** [vernesong/OpenClash#3452] [Q&A]
我不清楚这是不是bug，所以感觉发在讨论这里比较合适点。
就是启用fakeip之后qb里tracker会提示skipping tracker announce (unreachable)，我在网上找了很多方法，包括在qb设置里绑定pppoe端口、跳过https证书验证等，似乎都不行。而之前同样是fakeip下transmission连接就没有问题，不知道有没有人遇到同样的问题？
目前是r4s用的istores os系统，openclash是默认最新的dev内核

**Comment by ghost** [comment]
没用过istoresos，但应该是一个原因吧

是 bug

但是 qbt 的，有人反馈过，但官方没找到

你换回 4.2 及以前的版本，能正常一部分

或者查完机场 tos 后，决定是否把 `代理服务器` 打开（自建就没这顾虑了）

`类型` `覆写设置-常规设置` 里找 `主机` `127.0.0.1` `端口` 和 `类型` 对应 `验证-用户名&密码` 也在 `覆写设置` 最下面

pppoe 接口 + **关闭**`使用代理服务器进行用户连接` 只会走少量流量（根据种子数量和活跃度不同流量不同）（1000连接按1M/H算看呢不能接受，实际是小于的）

**Reply by kid-oh** [reply]
好的谢谢，后面我用回redir模式去了，现在跑去用passwall了，去面对另一个bug了（ipv6客户端上用不了）🤣

**Reply by dahuangshushu** [reply]
你好，能否描述的再详细一些，怎样设置openclash和qbittorrent能避免skipping tracker announce (unreachable)

**Reply by fouvromenth** [reply]
现在不是有geosite或ruleset的fake ip filter了吗？还需要这样设置吗？我现在换软件了，不清除是否能用。
上面的openclash的覆写设置中的内容，下方的是qbt中的
![1](https://github.com/user-attachments/assets/a3d907c6-7f79-4f74-8c95-46efce7178ab)

**Comment by werran2** [comment]
2025年了，还是没有解决这个bug

**Reply by werran2** [reply]
对了，我用的5.03

**Reply by mapledot** [reply]
我没有遇到这个问题，fakeip模式下qBittorrent可以正常使用

**Comment by yifai** [comment]
几年了，没遇到过这个问题，正常使用

**Comment by Rangtian** [comment]
未必是OpenClash的问题，我用的ImmortalWrt，一点问题都没有。我正在用的OpenClash模板维护者就明确表示：非 OpenWrt/ImmortalWrt 官方固件，特别是 iStore/kwrt 之流，免问。

![image](https://github.com/user-attachments/assets/3978da3d-e6c2-488f-8b29-930378cff724)
